src/auth/SignInComponent.tsx

```diff
@@ -34,7 +41,9 @@ function SignInComponent() {
         login();
         navigate("/");
       })
-      .catch((err) => console.error("Login faild:", err));
+      .catch(() => {
+        setError("Incorrect email or password.");
```

Comment:
It should only happen if error is 401. Other types of error should not display that message, but rather some generic error message.